config/lint/.markdown-lint.yaml

```diff
@@ -0,0 +1,77 @@
+# Start with default values
+# Ref: https://github.com/DavidAnson/markdownlint/blob/main/schema/.markdownlint.yaml
+default: true
+
+# MD003/heading-style : Heading style : https://github.com/DavidAnson/markdownlint/blob/main/doc/md003.md
+MD003:
+  # Heading style
+  style: "atx"
+
+# MD007/ul-indent : Unordered list indentation
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md007.md
+MD007:
+  indent: 4
+
+# MD013/line-length : Line length
+# https://github.com/DavidAnson/markdownlint/blob/main/doc/md013.md
+MD013:
+  # Number of characters
+  line_length: 80
+  # Do not apply line length rule to code blocks, tables or headings
+  # Links and images are excluded by default.
+  code_blocks: false
```

Comment:
I think this should be set to `true`

Reply:
Yes, if we want to enforce line length for code blocks as well. It might not always work tho. I'll set it to `true`.

Reply:
Done.